pkg/azurefile/utils.go

```diff
@@ -265,3 +275,47 @@ func replaceWithMap(str string, m map[string]string) string {
 	}
 	return str
 }
+
+func getAzcopyJob(dstFileshare string) (AzcopyJobState, string, error) {
+	cmdStr := fmt.Sprintf("azcopy jobs list | grep %s -B 3", dstFileshare)
+	out, err := exec.Command("sh", "-c", cmdStr).CombinedOutput()
+	if err != nil && err.Error() != "exit status 1" {
+		return AzcopyJobError, "", fmt.Errorf("couldn't list jobs in azcopy %v", err)
+	}
+	jobid := ""
+	jobSegments := strings.Split(string(out), "JobId: ")
```

Comment:
pls paste `azcopy jobs list` output in the comments, otherwise the code is very difficult to read.

Reply:
done